cobbler/actions/mkloaders.py

```diff
@@ -69,13 +79,22 @@ def run(
                 self.bootloaders_dir.joinpath("grub", bl_mod_dir),
             )
 
+    def create_directories(self):
+        """
+        Create the required directories so that this succeeds. If existing, do nothing.
+        """
+        if not self.bootloaders_dir.exists():
+            raise FileNotFoundError("Main bootloader directory not found! Please create it yourself!")
+
+        grub_dir = self.bootloaders_dir.joinpath("grub")
+        if not grub_dir.exists():
+            os.mkdir(grub_dir, 0o644)
```

Comment:
Pathlib can also create directories, I'd stick to one of `pathlib` / `os.path`
```suggestion
            grub_dir.mkdir(mode=0o644)
```
[Docs](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir)

Reply:
Will be applied to my next commit. Thanks!